docs/modules/position.md

```diff
@@ -0,0 +1,609 @@
+    function #bs.position:set_pos_x {scale:1}
+
+    ```
+:::
+:::{tab-item} y only
+**`#bs.position:set_pos_y {scale:<scaling>}`**
+
+Allows to place the entity at a precise y coordinate given by a score.
+
+Inputs
+
+:   (execution) `as <entities>`
+    : The entity to teleport.
+
+    (score) `@s bs.pos.y`
+    : The y coordinate to teleport to.
+
+    (macro variable) `scale`: double
+    : Scalar for the function's outputs.
+
+Outputs
+
+:   (state) Entity position
+    : The entity has been teleported to the y coordinate indicated by the score.
+
+Example
+
+:   Teleport to ~ 82 ~:
+
+    ```mcfunction
+    # Once
+    scoreboard players set @s bs.pos.y 82
+    function #bs.position:set_pos_y {scale:1}
+
+    ```
+:::
+:::{tab-item} z only
+**`#bs.position:set_pos_z {scale:<scaling>}`**
+
+Allows to place the entity at a precise z coordinate given by a score.
+
+Inputs
+
+:   (execution) `as <entities>`
+    : The entity to teleport.
+
+    (score) `@s bs.pos.z`
+    : The z coordinate to teleport to.
+
+    (macro variable) `scale`: double
+    : Scalar for the function's outputs.
+
+Outputs
+
+:   (state) Entity position
+    : The entity has been teleported to the z coordinate indicated by the score.
+
+Example
+
+:   Teleport to ~ ~ 65:
+
+    ```mcfunction
+    # Once
+    scoreboard players set @s bs.pos.z 65
+    function #bs.position:set_pos_z {scale:1}
+
+    ```
+:::
+::::
+
+> **Credits**: Aksiome, Leirof
+
+---
+
+### Set rotation
+
+::::{tab-set}
+:::{tab-item} h,v
+**`#bs.position:set_rot {scale:<scaling>}`**
+
+Allows to orientate the entity according to its scores.
+
+Inputs
+
+:   (execution) `as <entities>`
+    : The entity to teleport.
+
+    (score) `@s bs.rot.[h,v]`
+    : The rotation to teleport to.
+
+    (macro variable) `scale`: double
+    : Scalar for the function's outputs.
+
+Outputs
+
+:   (state) Entity rotation
+    : The entity has been teleported to the rotation indicated by the scores.
+:::
+:::{tab-item} h only
+**`#bs.position:set_rot_h {scale:<scaling>}`**
+
+Allows to orientate the entity according to its horizontal rotation score.
+
+Inputs
+
+:   (execution) `as <entities>`
+    : The entity to teleport.
+
+    (score) `@s bs.rot.h`
+    : The rotation to teleport to.
+
+    (macro variable) `scale`: double
+    : Scalar for the function's outputs.
+
+Outputs
+
+:   (state) Entity rotation
+    : The entity has been teleported to the rotation indicated by the horizontal score.
+:::
+:::{tab-item} v only
+**`#bs.position:set_rot_v {scale:<scaling>}`**
+
+Allows to orientate the entity according to its vertical rotation score.
+
+Inputs
+
+:   (execution) `as <entities>`
+    : The entity to teleport.
+
+    (score) `@s bs.rot.v`
+    : The rotation to teleport to.
+
+    (macro variable) `scale`: double
+    : Scalar for the function's outputs.
+
+Outputs
+
+:   (state) Entity rotation
+    : The entity has been teleported to the rotation indicated by the vertical score.
+:::
+::::
```

Comment:
Nothing happen (executed on a player)

Reply:
Work also fine, maybe an entity was missing when you tested it?